### PR TITLE
Adding PHP Unit Test Filtering

### PR DESCRIPTION
**Pull request by adamgoose**
Contribution Type: cosmetic
Primary Area: front-end

Description of change: Added the ability to filter PHPUnit Test Results by "All", "Passed", or "Failed". Defaults to "Failed". Screenshot here: http://d.enge.me/1itcH

### CODE REVIEW

**Comment by RobinMalfait** [comment]
+1 this!

**Comment by tvbeek** [comment]
+1

**Comment by flowl** [comment]
How did you even get PHPUnit running with PHPCI... there is still the parsing bug #674
No chance for me to get it working. Help would be appreciated :-)

**Comment by adamgoose** [comment]
@flowl I was having issues with it too, until I sent #726

File: public/assets/js/build-plugins/phpunit.js [hunk 1]
```diff
@@ -21,6 +21,20 @@ var phpunitPlugin = ActiveBuild.UiPlugin.extend({
...
+        $(document).on('click', '#phpunit-filter-all', function() {
+            $('#phpunit-data tbody tr').show();
+        });
+
+        $(document).on('click', '#phpunit-filter-pass', function() {
+            $('#phpunit-data tbody tr').hide();
+            $('#phpunit-data tbody tr.success').show();
+        });
+
+        $(document).on('click', '#phpunit-filter-fail', function() {
```
Comment (snapshotpl) [inline public/assets/js/build-plugins/phpunit.js]: Why you create 3 times the same object? Reuse it!

**Comment by mavimo** [comment]
@adamgoose please can you fix @snapshotpl note? I think can be fast :)

**Comment by dancryer** [comment]
@adamgoose This is great! :)

**Comment by dancryer** [comment]
@adamgoose That said, please could you rebase this onto the current master branch? I'm struggling to get it merged as-is.

Thanks

Dan

**Comment by mavimo** [comment]
@adamgoose do you have time to fix the small issue on JS side and rebase it? :smile:

**Pull request closed without merging**